aries-site/src/components/home/Highlights.js

```diff
@@ -44,16 +43,11 @@ const HighlightsLayout = () => {
                   />
                 )}
               </Box>
-              <Box fill>
-                <Text
-                  weight="bold"
-                  size="large"
-                  margin={{ top: 'small' }}
-                  color="text-strong"
-                >
+              <Box fill gap="xsmall">
+                <Heading level={3} margin={{ top: 'small', bottom: 'none' }}>
```

Comment:
Do we need both gap and margin specified? Can we eliminate one?

Reply:
Good call, I'll look to remove `margin` if possible.